src/libOpenImageIO/imagebufalgo_opencv.cpp

```diff
@@ -136,7 +137,182 @@ IplImage *
 ImageBufAlgo::to_IplImage (const ImageBuf &src)
 {
 #ifdef USE_OPENCV
-    return NULL;
+    ImageSpec spec = src.spec();
```

Comment:
I think this should be

```
    const ImageSpec &spec (spec.spec());
```

There's no reason to make a full copy of the spec, right? I don't think you intend to modify a local copy, you are just making a shorthand to refer to src.spec() in a less verbose way, and that's exactly what a `const&` is good for.